Tests/Marketplace/marketplace_services.py

```diff
@@ -389,14 +395,41 @@ def all_levels_dependencies(self):
         return self._all_levels_dependencies
 
     @property
-    def statistics_metadata(self):
-        return {
-            Metadata.DOWNLOADS: self.downloads_count,
-            Metadata.SEARCH_RANK: self._search_rank,
+    def update_metadata(self):
+        """
+        Returns a dictionary containing updated metadata fields.
+        This function updates the statistics_metadata fields (downloads, searchRank, tags, and integrations).
+        If is_metadata_updated is True, it also updates the fields that are not listed in PACK_METADATA_REQUIRE_RN_FIELDS.
+        Returns:
+            dict: Updated metadata fields.
+        """
+        update_statistics_metadata = {
+            Metadata.CREATED: self._create_date,
+            Metadata.UPDATED: self._update_date,
+            Metadata.DOWNLOADS: self._downloads_count,
             Metadata.TAGS: list(self._tags or []),
-            Metadata.INTEGRATIONS: self._related_integration_images
+            Metadata.SEARCH_RANK: self._search_rank,
+            Metadata.INTEGRATIONS: self._related_integration_images,
         }
 
+        update_metadata_fields = {}
+        if self.is_metadata_updated:
+            pack_metadata = self.pack_metadata
+            for field in pack_metadata:
+                if field not in PACK_METADATA_REQUIRE_RN_FIELDS:
+                    update_metadata_fields[field] = pack_metadata.get(field)
```

Comment:
```suggestion
            update_metadata_fields = {k: self.pack_metadata.get(k) for k in PACK_METADATA_REQUIRE_RN_FIELDS}
```

Reply:
it does not have the same meaning :see_no_evil:

Reply:
```suggestion
            update_metadata_fields = {k: self.pack_metadata.get(k) for k in self.pack_metadata if k not in PACK_METADATA_REQUIRE_RN_FIELDS}
```
Would this work?...

Reply:
yes, thanks!